<?xml version="1.0" ?>
<launch>

    <arg name="points_in" default="camera/depth_registered/points" />

    <node pkg="industrial_filters" name="test_avg_concat_node" type="concatenate_average" output="screen">
	<!--topic remapping-->
        <remap from="cloud_in" to="$(arg points_in)" />

	<param name="mls_search_radius" value="0.010" />
	<param name="num_images_concat" value="5" />

   </node>

</launch>
